### PR TITLE
cursory styling and formatting

### PR DESCRIPTION
**Pull request by mrcrymnd**

### CODE REVIEW

**Comment by nbuonin** [comment]
@mrcrymnd I think it would make sense to put what's in `css/custom.css` into a scss partials. That way we can make use of the benefits of the scss build, and it will make it easier to carry over to Django.

So for example lines 7 through 24 can go into a typography partial, and the other things can go into a layout partial.

File: scss/main.scss [hunk 1]
```diff
@@ -5,8 +5,10 @@
...
 // If you update Bootstrap be sure to remember to update the JS loaded from
 // Bootstrap's CDN.
-@import 'bootstrap-4.1.0/bootstrap';
 
-@import 'lib/cover';
+@import 'bootstrap-4.1.0/bootstrap';
 @import 'base/base';
-@import 'layouts/nav';
+
+//@import 'layouts/nav';
```
Comment (nbuonin) [inline scss/main.scss]: @mrcrymnd Looks great!

Question: did you intend to include `layouts/nav` and `layouts/graphwidget` here?  You have the files checked in but they aren't linked to anything.

File: scss/base/base.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+p a, li a, h3 a { color: #0e48a1; }
+
+.btn-primary { background-color: #0e48a1; }
+
+
+footer { margin: 3em 0em 3em 0em; }
+
+.embed-responsive {border: 1px solid #ced4da; border-radius: .25rem;}
```
Comment (nbuonin) [inline scss/base/base.scss]: This is the same as `graphwidget` do you want to keep it here?

File: scss/main.scss (hunk 1 above)
Comment (mrcrymnd) [reply scss/main.scss]: oops fixed thank you

File: scss/base/base.scss (hunk 2 above)
Comment (mrcrymnd) [reply scss/base/base.scss]: oops fixed thank you

**Comment by nbuonin** [comment]
@mrcrymnd 🤘

**Pull request merged**